tailwind.config.js

```diff
@@ -26,7 +26,14 @@ module.exports = {
     fontFamily: {
       sans: ["Roboto", "Helvetica", "Arial", "sans-serif"],
     },
-    extend: {},
+    extend: {
+      colors: {
+        textAccent: "var(--textColorAccent)",
+        textPrimary: "var(--textColorPrimary)",
+        textTertiary: "var(--textColorTertiary)",
+        textDimmed: "var(--textColorDimmed)",
```

Comment:
Do we still need these? As we consider using plain _tailwind_ class names more as _anti-pattern_ inside react components (when could be easily replaced with plain css alternative inside module style file). 

So using `.item {color: var(--textColorAccent);}` more preferable than using `tailwind` class `text-textAccent`. Wdyt?